workers/data_refinery_workers/processors/test_array_express.py

```diff
@@ -82,3 +103,18 @@ def test_affy_to_pcl_no_brainarray(self):
         self.assertEqual(ComputedFile.objects.all()[0].filename, 'GSM45588.PCL')
 
         os.remove(ComputedFile.objects.all()[0].absolute_file_path)
+
+    @tag("affymetrix", "huex")
```

Comment:
Do you want a test in here somewhere that makes sure things fail without the override?

Reply:
hmm I'm not sure how i would be able to test that since it reads from the file... would need to have another example that we don't want to override and should fail?

Reply:
I was imagining that you'd have the same file without the override & it would fail + the version with the override we would test for success. Maybe it's overkill for this case.